orchagent/pfcwdorch.cpp

```diff
@@ -480,6 +502,11 @@ void PfcWdSwOrch<DropHandler, ForwardHandler>::registerInWdDb(const Port& port,
 
         losslessTc.insert(i);
     }
```

Comment:
what if you get one lossless tc here, and later another lossless tc is added to the port, are you going to retry? I can see you only return false when there is no lossless tc.

Reply:
This is no processing difference w/ and w/o the PR change regarding the case of another lossless tc being added later to the port.